Gemfile

```diff
@@ -34,6 +34,6 @@ gem 'rake', :require => false
 gem 'facter', ENV['FACTER_GEM_VERSION'], :require => false, :groups => [:test]
 
 puppetversion = ENV['PUPPET_VERSION'] || '>= 6.0'
-gem 'puppet', puppetversion, :require => false, :groups => [:test]
+gem 'puppet', puppetversion, '<= 7.13.0', :require => false, :groups => [:test]
```

Comment:
Looks like the puppet-agent package (deb/rpm) is installed with the latest version…

There was a discussion on slack yesterday (not sure this link will work):
https://app.slack.com/client/T0AQJ2CTU/C0W298S9G/thread/C0W298S9G-1640810373.214300

@natemccurdy followed the white rabbit down to:
* https://github.com/puppetlabs/puppet/pull/8816
* https://github.com/puppetlabs/puppet/pull/8789
 
What we see here is one of two issues, a ticket was opened for it:
https://tickets.puppetlabs.com/browse/PUP-11402